### PR TITLE
CLOUDP-189436: add missing template validation. Fix broken templates

### PR DESCRIPTION
**Pull request by wtrocki**
Continuing work from #2020 to add validation and discover issues with the table templates.

### CODE REVIEW

File: internal/cli/atlas/dbusers/certs/list.go [hunk 1]
```diff
@@ -52,8 +52,8 @@ func (opts *ListOpts) Run() error {
 	return opts.Print(r)
 }
 
-var listTemplate = `ID USERNAME	SUBJECT CREATED AT{{range .Results}}
-{{.Id}}	{{.Username}}	{{.Subject}}	{{.CreatedAt}}{{end}}
```
Comment (wtrocki) [inline internal/cli/atlas/dbusers/certs/list.go:56]: Username was never returned (even originally)

**Review by andreaangiolillo — DISMISSED** [review]
LGTM 🎉

**Comment by andreaangiolillo** [comment]
for the failing doc test fail, make sure to run `make gen-docs` and to push all the changes

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Comment by wtrocki** [comment]
Thank you for a review.

**Pull request merged**